app/assets/javascripts/about.js

```diff
@@ -1,26 +1,28 @@
-(function (jQuery) {
-  jQuery.mark = {
-    jump: function (options) {
-      var defaults = {
-        selector: 'a.prev'
-      };
-      if (typeof options == 'string') defaults.selector = options;
-      var options = jQuery.extend(defaults, options);
-      return jQuery(options.selector).click(function (e) {
-        var jumpobj = jQuery(this);
-        var target = jumpobj.attr('href');
-        var thespeed = 1000;
-        var offset = jQuery(target).offset().top;
-        jQuery('html,body').animate({
-          scrollTop: offset
-        }, thespeed, 'swing')
-        e.preventDefault();
-      })
+$(document).on('page:change', function() {
+    (function (jQuery) {
```

Comment:
Nitpick but this is supposed to only be indented two spaces right? Also, do you think you can put a comment somewhere in this file saying what this is doing? For future people